app/lib/file_parse_service.rb

```diff
@@ -99,27 +99,25 @@ def self.run_parse_job(study_file, study, user, reparse: false, persist_on_fail:
           Rails.logger.info "Aborting parse of #{@study_file.name} as #{@study_file.file_type} in study #{@study.name}; not applicable"
         end
       when 'AnnData'
-        # gate ingest of AnnData using the feature flag 'ingest_anndata_file'
-        if do_anndata_file_ingest == true
+        # gate full ingest of an AnnData file using the feature flag 'ingest_anndata_file'
+        if do_anndata_file_ingest
           # Currently assuming "Happy Path" and so the AnnData file will have clustering data
           # extract and parse clustering data
           job = IngestJob.new(study: study, study_file: study_file, user: user, action: :ingest_anndata, reparse: reparse,
           persist_on_fail: persist_on_fail)
-          job.delay.push_remote_and_launch_ingest
-
           # Future consideration about whether to do all in one job likely in (SCP-4754)
           # TODO extract and parse Metadata (SCP-4708)
           # TODO extract and parse Processed Exp Data (SCP-4709)
           # TODO extract and parse Raw Exp Data (SCP-4710)
         else
-          Rails.logger.info "Aborting parse of AnnData file #{study_file.name} due to feature flag being #{do_anndata_file_ingest}"
+          # launch an ingest job for parsing a reference AnnData file
+          job = IngestJob.new(study: study, study_file: study_file, user: user, action: :ingest_anndata_reference, reparse: reparse,
+          persist_on_fail: persist_on_fail)
```

Comment:
This is a good example of how specifying values that have names matching keys degrades readability.  

I suggest:

```suggestion
          job = IngestJob.new(study:, study_file:, user:, action: :ingest_anndata_reference, reparse:, persist_on_fail:)
```

Benefits:
* Faster to discern most salient part of assignment -- the part that's not implied by the key
* Better visual chunking due to assignment fitting on one line, rather than filling two
* Truly leverage one of the biggest features in Ruby 3.1, rather than our quite constrained recent usage
* Easier jump between languages, e.g. SCP JS is replete with its equivalent of "mixed" Hash literal style

I'm unaware of any costs not already incurred in our JS, where such mixing has seemed successful.

If the above persuades folks, I'd propose we [remove this line in our Rubocop lint rules](https://github.com/broadinstitute/single_cell_portal_core/blob/development/.rubocop.yml#L58), and gradually apply Ruby 3.1 shorthand syntax wherever it applies.

Reply:
I'm ambivalent about the rubocop change - personally I think consistency matters, but it does allow for better concision so I'm fine if the rest of the team wants to make that change.

Reply:
4ecb737